bower.json

```diff
@@ -2,7 +2,7 @@
   "name"          : "backbone",
   "main"          : "backbone.js",
   "dependencies"  : {
-    "underscore"  : ">=1.8.3"
+    "underscore"  : ">=1.9.0"
```

Comment:
did something change that makes 1.8.3 not still valid?

Reply:
I know he updated Underscore recently, and there are still bugs over in that being looked at. I was just going over unit test code when you replied! (too tired to finish that up tonight)

Reply:
Why does it matter if underscore was updated.  Currently this lib support 1.8.3 and 1.9.x etc.. upping this to 1.9.0 removes support for 1.8.3 unnecessarily.  Unless you've used something from 1.9 that isn't in 1.8.3?